cmd/cluster-node-tuning-operator/main.go

```diff
@@ -194,6 +204,46 @@ func operatorRun() {
 		if err = (&performancev2.PerformanceProfile{}).SetupWebhookWithManager(mgr); err != nil {
 			klog.Exitf("unable to create PerformanceProfile v2 webhook: %v", err)
 		}
+	} else {
+		// Hypershift configuration
+		restConfig, err := ntoclient.GetInClusterConfig()
+		if err != nil {
+			klog.Exitf("unable to create get InClusterConfiguration while creating PerformanceProfile controller: %v", err)
+		}
+
+		fOps := func(opts *cluster.Options) {
+			operatorNamespace := config.OperatorNamespace()
+			opts.Cache.Namespaces = []string{operatorNamespace}
+			opts.Scheme = mgr.GetScheme()
+			opts.MapperProvider = func(c *rest.Config, httpClient *http.Client) (meta.RESTMapper, error) {
+				return mgr.GetRESTMapper(), nil
+			}
+			opts.NewClient = func(config *rest.Config, options client.Options) (client.Client, error) {
+				c, err := client.New(config, options)
+				if err != nil {
+					return nil, err
+				}
+				return hypershift.NewControlPlaneClient(c, operatorNamespace), nil
+			}
+		}
+		managementCluster, err := cluster.New(restConfig, fOps)
+		if err != nil {
+			klog.Exitf("unable to create ManagementCluster while creating PerformanceProfile controller : %v", err)
+		}
+
+		if err := mgr.Add(managementCluster); err != nil {
+			klog.Exitf("unable to add ManagementCluster to manger while creating PerformanceProfile controller : %v", err)
```

Comment:
Can we have some consistency with logging?  Sometimes we say:

```
message<space>: %
```
sometimes
```
message: %
```

I vote for the second.  Ideally across the entire PR, not just in `main.go`